BASE/src/Microsoft.ApplicationInsights/Channel/InMemoryChannel.cs

```diff
@@ -96,6 +95,20 @@ public TimeSpan SendingInterval
             }
         }
 
+        /// <summary>
+        /// Gets or sets the <see cref="CredentialEnvelope"/> which is used for AAD.
+        /// FOR INTERNAL USE. Customers should use <see cref="TelemetryConfiguration.SetAzureTokenCredential"/> instead.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="ISupportCredentialEnvelope.CredentialEnvelope"/> on <see cref="InMemoryChannel"/> sets <see cref="InMemoryTransmitter.CredentialEnvelope"/> 
+        /// which is used to set <see cref="Transmission.CredentialEnvelope"/> just before calling <see cref="Transmission.SendAsync"/>.
+        /// </remarks>
+        CredentialEnvelope ISupportCredentialEnvelope.CredentialEnvelope
```

Comment:
Access modifier is missing here. If the plan is to mark this method as internal, we could simplify the summary section removing "For internal use".

Reply:
So the reason this looks weird is because this is an [explicitly implemented interface](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation).
Explicitly Implemented interfaces cannot have access modifiers and can only be accessed via casting back to the interface.

The only way to avoid this is to make the new interface public.